test/mochaSetup.js

```diff
@@ -1,25 +1,21 @@
 /* global global */
 
-require('babel/register')({
```

Comment:
This is now done automatically. If not removed, Babel will complain that polyfill is inserted twice.